docs/topics/mta-web-seeding-instances.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-seeding-instances_{context}"]
+= Seeding instances
+
+If you are a project architect you can configure the instances key parameters in the Controls window, before migration. The parameters can be added and edited as needed.
+The following parameters define applications, individuals, teams, verticals or areas within an organization affected or participating in the migration:
+
+* Stakeholders
+* Stakeholder groups
+* Job functions
+* Business services
+* Tag types
+* Tags
+
+You can create and configure an instance in any order. However, the suggested order below is the most efficient for creating stakeholders and tags.
+
+Stakeholders:
+
+. Create stakeholder groups
+. Create job functions
+. Create stakeholders
+
+Tags:
+
+. Create tag types
+. Create tags
+
+Stakeholders and defined by:
+
+* Email
+* Name
+* Job function
+* Stakeholder groups
+
+== Creating a new stakeholder group
+
+You can follow these steps to create a new stakeholder group. There are no default stakeholder groups defined.
+
+.Procedure
+
+. In *Developer* view, click *Controls*.
+. Click *Stakeholder groups*.
+. Click *Create new*.
+. Enter the following information:
+** Name
+** Description
+** Member(s)
+. Click *Create*.
+
+== Creating a new job function
+
+{ProductName} ({ProductShortName}) uses the job function attribute to classify stakeholders and provides a list of default values that can be expanded.
+
+You can follow these steps to create a new job function, which is not in the default list.
+
+.Procedure
+
+. In *Developer* view, click *Controls*.
+. Click *Job functions*.
+. Click *Create new*.
+. Enter a job function title in the *Name* text box.
+. Click *Create*.
+
+== Creating a new stakeholder
+
+You can follow these steps to create a new migration project stakeholder.
+
+.Procedure
+
+. In *Developer* view, click *Controls*.
+. Click *Stakeholders*.
+. Click *Create new*.
+. Enter the following information:
+** Email
+** Name
+** Job function - custom functions can be created
```

Comment:
@CarmiWisemon 
"custom functions can be created" - why is it here? Creating new job functions has already been described above.